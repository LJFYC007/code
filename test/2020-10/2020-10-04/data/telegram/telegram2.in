10
15 7 11 13 11 8 7 13 10 10 14 16 14 11 10 16 13 6 8 6 3 2 8 5 12 10 7 6 12 9 12 9 8 14 11 7 6 12 9 3 9 6 8 5 11 