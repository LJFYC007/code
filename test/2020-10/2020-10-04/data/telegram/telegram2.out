1
1 1 2 4 5 5 6 7 7 9 
